### PR TITLE
Release/2.0.9

### PR DESCRIPTION
**Pull request by istreeter**

### CODE REVIEW

File: project/BuildSettings.scala [hunk 1]
```diff
@@ -45,14 +45,6 @@ object BuildSettings {
...
     Seq(
       Docker / packageName := "elasticsearch-loader",
-      dockerRepository := Some("snowplow"),
-      dockerBaseImage := "eclipse-temurin:11-jre-focal",
-      Docker / maintainer := "Snowplow Analytics Ltd. <support@snowplowanalytics.com>",
-      Docker / daemonUser := "daemon",
-      dockerCmd := Seq("--help"),
-      dockerUpdateLatest := true,
-      Docker / daemonUserUid := None,
-      Docker / defaultLinuxInstallLocation := "/opt/snowplow"
```
Comment (benjben) [inline project/BuildSettings.scala:55]: Love that!

**Review by benjben — APPROVED** [review]

File: elasticsearch/src/main/scala/com/snowplowanalytics/stream/loader/clients/ElasticsearchBulkSender.scala [hunk 2]
```diff
@@ -101,7 +100,7 @@ class ElasticsearchBulkSender(
    * This BulkHandler is added to change endpoints of bulk api request to add
    * document type to them. This change is made in order to continue to support ES 6.x
    */
-  implicit object CustomBulkHandler extends Handler[BulkRequest, BulkResponse] with Logging {
+  implicit object CustomBulkHandler extends Handler[BulkRequest, BulkResponse] {
```
Comment (benjben) [inline elasticsearch/src/main/scala/com/snowplowanalytics/stream/loader/clients/ElasticsearchBulkSender.scala:103]: Don't we care about logging?

File: project/Dependencies.scala [hunk 3]
```diff
@@ -17,7 +17,7 @@ object Dependencies {
   object V {
     // Java
     val config           = "1.4.1"
-    val slf4j            = "1.7.32"
+    val slf4j            = "2.0.6"
```
Comment (benjben) [inline project/Dependencies.scala:20]: Have you testing that logging still works after bumping the major? I think that I've already had an issue with that

File: elasticsearch/src/main/scala/com/snowplowanalytics/stream/loader/clients/ElasticsearchBulkSender.scala (hunk 2 above)
Comment (istreeter) [reply elasticsearch/src/main/scala/com/snowplowanalytics/stream/loader/clients/ElasticsearchBulkSender.scala:103]: The `Logging` trait got removed from elastic4s [in this commit](https://github.com/sksamuel/elastic4s/commit/3404e5ab13c0d0c75800ca66c84cdaf877867041) and it got replaced with using slf4j directly.  So this line change is necessary and safe.  It does not change what gets logged.

File: project/Dependencies.scala (hunk 3 above)
Comment (istreeter) [reply project/Dependencies.scala:20]: Yes it is tested.  Logging still works.

**Pull request merged**